modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -87,215 +113,193 @@ to the majority of the brokers responsible for the partition in the cluster. As
 soon as the fsync call is complete, the message is considered acknowledged and
 is made visible to readers.
 
-NOTE: This parameter has an important distinction compared to Kafka's behavior. In
+NOTE: This property has an important distinction compared to Kafka's behavior. In
 Kafka, a message is considered acknowledged without the requirement that it has
-been fsynced. Messages that have not been flushed to disk will be lost in the
+been fsynced. Messages that have not been flushed to disk are lost in the
 event of a broker crash. So when using `acks=all`, the Redpanda default
 configuration is more resilient than Kafka's.
```

Comment:
I think we an add a reference to the write caching section.. 

write caching is a relaxed mode of acks=all that trades off durability for extra performance....